File: POOJAVA/src/clasesPOO/Area.java
package clasesPOO;

public class Area {
    String idArea;
    String nArea;
    String ubicacion;
    public Area(String idA,String nA)
    {
        establecerDatos(idA,nA);
    }
    public String ObtenerIdArea()
    {
       return idArea;
    }
    public String ObtenerArea()
    {
       return nArea;
    }
    void  establecerDatos(String idAr,String nAr)
    {
        idArea=idAr;
        nArea=nAr;
    }
}

File: POOPHP/app/Model/Area.php
<?php
namespace App\Model;
class Area
{
   public $idArea="";
   public $nUser="";
   public $nArea="";
   public $ubicacion="";
   public function __construct(String $idA,String $nA)
    {
        $this->establecerDatos($idA,$nA);
    }
    public function ObtenerIdArea()
    {
        return $this->idArea;
    }
    public function ObtenerArea()
    {
        return $this->nArea;
    }
    public function establecerDatos(String $idAr,String $nAr)
    {
        $this->idArea=$idAr;
        $this->nArea=$nAr;
    }
}
?>